apps/mobile/src/screens/CreatePost/FormPost/index.tsx

```diff
@@ -46,48 +53,105 @@ export const FormCreatePost: React.FC = () => {
     setImage(pickerResult.assets[0]);
   };
 
+  const handleVideoSelect = async () => {
+    const result = await ImagePicker.launchImageLibraryAsync({
+      mediaTypes: ImagePicker.MediaTypeOptions.Videos,
+      allowsEditing: false,
+      selectionLimit: 1,
+      exif: false,
+      quality: 1,
+    });
+
+    if (!result.canceled) {
+      const asset = result.assets[0];
+      setVideo(asset);
+    }
+  };
+
   const handleSendNote = async () => {
     if (!note || note?.trim()?.length == 0) {
```

Comment:
Can you add the check of if(!note && !image && !video). Because we can send a Video/Image without a note now